style.css

```diff
@@ -0,0 +1,273 @@
+:root {
+    --primary-main: #0067fc;
+    --primary-light: #f0f8ff;
+    --primary-dark: #0048cd;
+    --white-main: #fff;
+
+    --font-size-extra-small: 0.75rem;
+    --font-size-small: 0.875rem; 
+    --font-size-medium: 1rem; 
+    --font-size-large: 1.5rem;
+
+    --size-1: 0.25rem;
+    --size-2: 0.5rem; 
+    --size-3: 1rem; 
+    --size-4: 2rem; 
+
+    --transition-duration: 0.3s;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+.container {
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    position: sticky;
+    height: 65px;
+    top: 0;
+    left: 0;
+    background-color: var(--white-main);
+    padding-right: var(--size-2);
+}
+
+.header-container {
+    display: flex;
+    justify-content: space-between;
+}
+
+header .logo {
+    width: 6.25rem;
+}
+
+.button {
+    cursor: pointer;
+    border: none;
+    background: none;
+    height: 2.1rem;
+    font-size: var(--font-size-extra-small);
+    color: var(--white-main);
+    background-color: var(--primary-main);
+    border-radius: var(--size-1);
+    transition: color var(--transition-duration);
+}
+
+.button:hover {
+    color: var(--primary-main);
+    background-color: var(--white-main);
+    border: 1px solid var(--primary-main);
+}
+
+nav {
+    display: flex;
+    align-items: center;
+    gap: var(--size-2);
+}
+
+.nav-link {
+    min-width: 4.5rem;
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.info-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: var(--size-2);
+    padding: var(--size-4);
+    text-align: center;
+    background-color: var(--primary-dark);
+    color: var(--white-main);
+}
+
+.info-section h1 {
+    font-size: var(--font-size-large);
+}
+
+.info-section p {
+    font-size: var(--font-size-small);
+    max-width: 500px;
+}
+
+.carousel-section {
+    height: 220px;
+    padding: var(--size-3);
+    background-color: var(--primary-light);
+}
+
+.carousel-container {
+    height: 100%;
+    display: flex;
+    justify-content: space-between;
+}
+
+.carousel-display {
+    width: 80%;
+    max-width: 500px;
+    display: flex;
+    background-image: url('https://images.unsplash.com/photo-1532298229144-0ec0c57515c7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1122&q=80');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.prev, .next {
+    min-width: 44px;
+    background: none;
+    border: none;
+    font-size: 2.5rem;
+}
+
+.categories-section {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    justify-content: center;
+    gap: var(--size-2);
+    padding: var(--size-4) 2rem;
+}
```

Comment:
Consider if it would be useful to extract these styles to a separate file. To keep the concerns separate (don't forget the media queries). For example:

```
// styles.css
@import 'base.css' // css reset, global styles
@import 'layout.css' // header / footer
@import 'carousel.css' // component styles
@import 'utilities.css' // .container class for example.
```

 https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Organizing#tips_to_keep_your_css_tidy